crates/modelardb_server/src/storage/data_transfer.rs

```diff
@@ -118,14 +103,13 @@ impl DataTransfer {
             .append(initial_disk_space as isize, true);
 
         // Check if data should be transferred immediately.
-        for table_name_column_index_size_in_bytes in compressed_files.iter() {
-            let table_name = &table_name_column_index_size_in_bytes.key().0;
-            let column_index = table_name_column_index_size_in_bytes.key().1;
-            let size_in_bytes = table_name_column_index_size_in_bytes.value();
+        for table_name_field_column_index_size_in_bytes in table_size_in_bytes.iter() {
```

Comment:
Maybe change to something like `for table_name_size_in_bytes in`.

Reply:
Fixed.